plugins/schedule/base/lib/iCal/kSchedulingICalEvent.php

```diff
@@ -249,7 +257,6 @@ public static function fromObject(KalturaScheduleEvent $event)
 		}
 
 		$resources = ScheduleEventResourcePeer::retrieveByEventId($event->id);
-		$resourceIds = array();

```

Comment:
I think that child resources should replace the parent resources if there are any, not to be added to the parent resources, but it should be decided by Renan.